Gemfile

```diff
@@ -98,6 +98,8 @@ gem 'gon', '~> 4.0'
 # don't require by default, instead load on-demand when actually configured
 gem 'airbrake', '~> 4.1.0', require: false
 
+gem 'transactional_lock', git: 'https://github.com/finnlabs/transactional_lock.git', branch: 'master'
```

Comment:
Okay, after some back and forth, I've now left the gem where it was, because it does not depend on any specific database backend itself, there is no gem dependency.

It will (as of today) only work correctly for a MySQL backend, but this is no design choice, but rather a limitation of the implementation (of the gem).